src/script/event/WebSocketService.js

```diff
@@ -117,13 +117,19 @@ z.event.WebSocketService = class WebSocketService {
 
       this.socket.onmessage = event => {
         if (this._pingHasExperiencedSuspiciousInactivity()) {
-          const secondsSinceLastPing = (Date.now() - this.lastPingTime) / z.util.TimeUtil.UNITS_IN_MILLIS.SECOND;
-          this.logger.warn(`Message received but ping was inactive for "${secondsSinceLastPing}" sec, reconnecting.`);
+          const secondsSinceLastPong = (Date.now() - this.lastPongTime) / z.util.TimeUtil.UNITS_IN_MILLIS.SECOND;
+          this.logger.warn(`Message received but there was no pong for "${secondsSinceLastPong}" sec, reconnecting.`);
           return this.reconnect(WebSocketService.CHANGE_TRIGGER.LONG_INACTIVITY);
         }
         if (event.data instanceof Blob) {
           const blobReader = new FileReader();
-          blobReader.onload = () => onNotification(JSON.parse(blobReader.result));
+          blobReader.onload = () => {
+            if (blobReader.result === 'pong') {
```

Comment:
Although the idea of listening to `pong` replies instead of sent `ping`, it doesn't seem that the backend is sending any `pong` message. 
Are you sure this condition is met at some point?

Reply:
Of course, see [discussion](https://github.com/wireapp/wire-server/issues/560#issuecomment-458139018) and [implementation](https://github.com/wireapp/wire-server/pull/561) 🙂

Reply:
The backend part is already on production as per last comment in the discussion thread 🙂 

> @maximbaz now this ticket is legitimately closed, and #561 is in production.